go/parquet/LICENSE.txt

```diff
@@ -0,0 +1,1987 @@
+
```

Comment:
I think these overlap with the top level license.txt, we should probably consolidate them there?  Or is there a reason to include them here?

Reply:
I believe that including it here is useful so the license gets picked up by pkg.go.dev? I know that the arrow go module also includes a LICENSE.txt in it's directory even though it overlaps with the top level one, so i was following that pattern.

Reply:
yeah, looking back it seems like we needed to copy to make this usable.  I think there is an open item for go to support symlinks.